### ISSUE TITLE
Enable auto-reloading for all config file settings

### ISSUE THREAD

**Issue by Baggykiin** [Baggykiin/pass-winmenu#3]
Low priority.

**Comment by Baggykiin** [comment]
Postponed indefinitely. Would require quite a lot of code to be rewritten for a rather small benefit.

**Comment by AeliusSaionji** [comment]
Perhaps a simpler solution would be to have some way to quickly restart the process. Tweaking the visual settings is currently an inelegant process.

Only allow one instance of the process to run at a time? A "restart" entry in the context menu? There are a few ways to smooth this out.

**Comment by Baggykiin** [comment]
True, I forgot I initially wanted this because it makes changing the visual settings a lot simpler. I don't think it's really needed for the other settings, so I'll have to think about how this could best be implemented.
